desktop/apps/auction2/actions.js

```diff
@@ -2,14 +2,17 @@ import metaphysics from '../../../lib/metaphysics.coffee'
 import { filterQuery } from './filter_query'
 
 // Action types
+export const TOGGLE_FETCHING_ARTWORKS = 'TOGGLE_FETCHING_ARTWORKS'
```

Comment:
@sweir27 - Rather than `TOGGLE_FETCHING_ARTWORKS` why not `GET_ARTWORKS_REQUEST` and then, later, if there's an error or success, you can dispatch `GET_ARTWORKS_FAILURE` or `GET_ARTWORKS_SUCCESS`? In each of those reducers you can toggle the preloader depending on the actual, overall state of the request, and it dually serves as a self-documenting action.